doc/formats.md

```diff
@@ -564,6 +565,59 @@ $ fq -d msgpack torepr file.msgpack
 ### References
 - https://github.com/msgpack/msgpack/blob/master/spec.md
 
```

Comment:
This part of `formats.md` is auto generated when running `make doc` what i meant was to put documentation and examples for each postgres format into files `format/postgres/<format-name>.md` that way they will be automatically be included in `formats.md` and it will also be used by the cli `fq -h <format-name>` or `help(<format-ame>)`. See `mp4.md` for example, probably good to keep the markdown simple, the cli markdown render is not very smart.

Reply:
I add umbeded *.md to formats and add postgres to format.md with code generation.